### ISSUE TITLE
Wearable APK

### ISSUE THREAD

**Issue by jordanmichaelrushing** [omkarmoghe/Pokemap#25]
Not an Android dev so I can only ask. Is the wearable apk also being dev'ed for this by anyone? If not, I've never made an android app (Rails dev), but could take a swing at adding on wearable apk functionality?

**Comment by Mahonster** [comment]
I've never introduced wearable support in my previous apps, but it seems like we might be a little ways away from that. :)

**Comment by jordanmichaelrushing** [comment]
Oh I'm sure it's no where near close based on the readme thus far (haven't even looked at the code since I don't speak java haha). Just wanted to see if this was something that might be able to be added to the roadmap.
Would be awesome to quickly glance at my watch to see nearby instead of switching between this app and the PoGp app.

**Comment by fess89** [comment]
As far as I can see, it is not yet being developed by anyone, but it is totally feasible and would be cool to do. Perhaps we could work on that after achieving base functionality.
BTW, I've created a Slack channel for the project: https://pokemap-android.slack.com, feel free to join (it should accept anyone with a Gmail account)

**Comment by omkarmoghe** [comment]
yeah would be dope, but might be hard until we get basic pokemon to show up

**Comment by s7092910** [comment]
@fess89 You need to invite people to join your slack. They can't just join it.

**Comment by omkarmoghe** [comment]
@s7092910 we've reached the limit on invites. until more of the invited people join, we cant add any more. :/

**Comment by psamsonov** [comment]
Wearables would be cool and all, but the Android watch isn't super popular. I was thinking of a background service that would send a notification when you're right on top of (or close enough to) a Pokemon you want, so that you don't have to use that ridiculous "battery saver" mode in the main app.

**Comment by omkarmoghe** [comment]
@psamsonov dope idea!

**Comment by JGreenlee** [comment]
I'm afraid they would get butthurt about sending notifications to a wearable device.

The $15 Pokemon Go Plus does exactly that, so it'd like taking money directly from their hands.

**Comment by jLynx** [comment]
If we end up doing this I will actually buy an android wear device just for this! This would be awesome!

**Comment by axsddlr** [comment]
@mell0 Meh. Not everyone has money for a wearable device nor do people even know about these API apps

**Comment by axsddlr** [comment]
You can integrate wearable notification straight from app. No need for separate apk

**Comment by jLynx** [comment]
I will take a look into this soon

**Comment by vidarr101** [comment]
I am working on a general notification for the app that tells you whats in the area and how far, wouldnt be too hard to make it notify a wearable too.

**Comment by Giggitybyte** [comment]
I figure this is the best place to ask:
Have any of you looked into some type of notification system for the Pebble smartwatches?
